app/ui/src/main/res/layout/folder_list_item.xml

```diff
@@ -7,21 +7,22 @@
     android:layout_height="wrap_content"
     android:gravity="center_vertical"
     android:minHeight="?android:attr/listPreferredItemHeight"
-    android:orientation="horizontal">
+    android:orientation="horizontal"
+    android:padding="8dp">
```

Comment:
The padding moves the folder icon too far to the right. You'll have to reduce the margin for the `ImageView` in order for the folder names to line up with the toolbar title.